.github/workflows/build-make-check.yml

```diff
@@ -0,0 +1,28 @@
+name: "Build with warnings and make check"
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  makecheck:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      id: dependencies
+      uses: ./.github/actions/install-dependencies
+
+    - name: Configure sssd
+      uses: ./.github/actions/configure
+
+    - name: Build with warnings and run make check
+      working-directory: x86_64
+      run: |
+        source ../contrib/fedora/bashrc_sssd
+        SSS_WERROR=-Werror chmake
```

Comment:
I think that we should provide the same behaviour locally and in the upstream CI checks. So, if we are to treat all warnings as errors in the CI tool, then local builds should do the same. There's already a line in `bashrc_sssd` file that could be uncommented to do this.

Reply:
just to be clear, do you propose to add a commit which uncomments the SSS_WERROR line in `bashrc_sssd` ?

Reply:
You could do amend the previous commit, but yes, in the end I'd like to uncomment the SSS_WERROR line in `bashrc_sssd`.

Reply:
Updated with another commit, though i'm not sure if this will break anything like QE automation scripts, etc.

Reply:
What do you mean? Is QE using this build target for anything?

Reply:
I think it is probably okay, I was just not sure if anyone may be using 'chmake' and expecting warnings to be ignored, not treated as errors.